Library/Formula/slib.rb

```diff
@@ -0,0 +1,44 @@
+class SchemeInstalled < Requirement
+  fatal true
+
```

Comment:
Can we add a `default_formula` here so it'll default to one of these if none are installed?